### DISCUSSION TITLE
how do I install OpenZeppelin on the Foundry branch?

### DISCUSSION THREAD

**Discussion by austintgriffith** [scaffold-eth/scaffold-eth-2#476] [Ideas]
I can't figure out how to install OZ

import "@openzeppelin/contracts/access/Ownable.sol";

isn't working even after 

yarn add @openzeppelin/contracts

what am I doing wrong? could we provide this out of the box for people or make it really easy for them to figure out

**Comment by technophile-04** [comment]
Yeah, we missed adding Openzeppelin out of the box in `se-2-foundry` :(  ...... but after #475  OpenZeppelin should come out box 🙌

---

To manually add libraries in foundry you need to do `forge install GhithubOrgName/repoName` and then add remappings to use it as `import` in solidity 

For eg : 

```
cd packages/foundry && forge install OpenZeppelin/openzeppelin-contracts
```
